src/libs/fileDownload/FileUtils.ts

```diff
@@ -156,58 +145,61 @@ function appendTimeToFileName(fileName) {
 
 /**
  * Reads a locally uploaded file
- *
- * @param {String} path - the blob url of the locally uplodaded file
- * @param {String} fileName
- * @param {Function} onSuccess
- * @param {Function} onFailure
- *
- * @returns {Promise}
+ * @param path - the blob url of the locally uploaded file
```

Comment:
Why were types removed from these 2 `@param` doc lines?

Reply:
Interesting, that's the opposite of what we do in our several typed languages (PHP, C++, Java)

Reply:
I'm confused here. C++ has the same recommendation -- to remove documentation that's redundant with what the code says:
https://github.com/Expensify/Style-Guides/blob/main/cpp.md#documentation

Reply:
There is no reason to specifying the types of the params in the comments anymore, because now with TS we specify the types in the function signature itself, that's why we are removing.

Reply:
Makes sense, I agree, thanks!